docs/ref-conflict-server.md

```diff
@@ -1,30 +1,34 @@
 # Server side conflict resolution
```

Comment:
I think we should provide quick explanation as to exactly _what_ conflict resolution is in this context. Obvious for most but I think it could help based on what we discussed with some community members. Even just a sentence on the problem we are trying to solve would cover it.